crates/eigen-cli/src/lib.rs

```diff
@@ -85,6 +124,54 @@ pub fn execute_egnkey_subcommand(subcommand: EigenKeyCommand) -> Result<(), Eige
             println!("{}", operator_id);
             Ok(())
         }
+
+        EigenKeyCommand::BlsConvert {
+            key_type,
+            secret_key,
+            output_path,
+            password,
+        } => {
+            BlsKeystore::from(key_type).new_keystore(
+                secret_key,
+                output_path,
+                password.as_deref(),
+            )?;
+            Ok(())
+        }
+        EigenKeyCommand::CreateNewMnemonic { language } => {
+            let mnemonic = match language {
+                MnemonicLanguage::English => {
+                    Mnemonic::get_mnemonic_without_word_path(ENGLISH_WORD_LIST, None).unwrap()
+                }
+                MnemonicLanguage::ChineseSimplified => {
+                    Mnemonic::get_mnemonic_without_word_path(CHINESE_SIMPLIFIED_WORD_LIST, None)
+                        .unwrap()
+                }
+                MnemonicLanguage::ChineseTraditional => {
+                    Mnemonic::get_mnemonic_without_word_path(CHINESE_TRADITIONAL_WORD_LIST, None)
+                        .unwrap()
+                }
+                MnemonicLanguage::Czech => {
+                    Mnemonic::get_mnemonic_without_word_path(CZECH_WORD_LIST, None).unwrap()
+                }
+                MnemonicLanguage::Italian => {
+                    Mnemonic::get_mnemonic_without_word_path(ITALIAN_WORD_LIST, None).unwrap()
+                }
+                MnemonicLanguage::Korean => {
+                    Mnemonic::get_mnemonic_without_word_path(KOREAN_WORD_LIST, None).unwrap()
+                }
+                MnemonicLanguage::Portuguese => {
+                    Mnemonic::get_mnemonic_without_word_path(PORTUGUESE_WORD_LIST, None).unwrap()
+                }
+                MnemonicLanguage::Spanish => {
+                    Mnemonic::get_mnemonic_without_word_path(SPANISH_WORD_LIST, None).unwrap()
+                }
+            };
+
+            println!("New mnemonic generated : {}", mnemonic);
+            println!("{}", "Please store it safely!".red().bold());
```

Comment:
probably needs removing

Reply:
it is deliberately added as a print to show in ci